src/tools/sssctl/sssctl_config.c

```diff
@@ -64,6 +64,9 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     /* Open config file */
     ret = sss_ini_config_file_open(init_data, SSSD_CONFIG_FILE);
     if (ret != EOK) {
+        if (ret == ENOENT) {
```

Comment:
I wanted the DEBUG message to be printed in both cases. Do you prefer the DEBUG message to be dropped in case of ENOENT (and print only ERROR)?